hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+  This release fixes a bug (:issue:`2166`) where a Unicode character info
+  cache file was generated but never used on subsequent test runs, causing tests
+  to run more slowly than they should have.
+
+  Thanks to Robert Knight for this bugfix!
```

Comment:
De-dent this text, and we'll be ready to merge!

(and released on PyPI ~30 minutes after that)

Reply:
Done